src/app/services/data-service.service.ts

```diff
@@ -1,14 +1,18 @@
 import { Injectable } from '@angular/core';
 import { BehaviorSubject } from 'rxjs';
+import { HttpClient } from '@angular/common/http';
+import { Observable } from 'rxjs';
+import { PeriodicElement } from '../equipment-car/equipment-car.component';
 
 @Injectable({
   providedIn: 'root',
 })
 export class DataServiceService {
   private homeData: any[] = [];
   private selectedCarData = new BehaviorSubject<any>(null);
+  private apiUrl = 'http://localhost:3000/elements';
```

Comment:
na API url najlepiej zrobi osobny plik, ponieważ ono będzie takie samo, dla całej aplikacji. Zrób sobie jakiś wspólny pliki `commons.ts` i tam wrzuć i wyeskportuj 

```
const API_URL = "http://localhost:3000"
```

i następnie dodasz tylko, z którego endpointu chcesz pobrać dane
return this.http.get<PeriodicElement[]>(API_URL+"/elements");

W przyszłości takie dane, jak API URL dobrze trzymać w env variables: https://www.digitalocean.com/community/tutorials/angular-environment-variables